projects/canopy/src/lib/card/docs/promotions/guide.stories.mdx

```diff
@@ -0,0 +1,513 @@
+          <a href="#">{{ buttonText }}</a>
+        </lg-card-content>
+      </lg-card>
+    </div>
+    <div lgColSm="12" lgColLg="4">
+      <lg-card
+        lgShadow
+        lgMarginBottom="lg"
+        variant="promotion"
+        [orientation]="{ sm: 'vertical', md: 'horizontal', lg: 'vertical' }">
+        <lg-card-hero-img>
+          <lg-brand-icon [name]="iconName" size="xs"></lg-brand-icon>
+        </lg-card-hero-img>
+        <lg-card-content>
+          <h3 lgMarginBottom="xxs" class="lg-font--expressive">{{ title }}</h3>
+          <p lgMarginBottom="lg">{{ text }}</p>
+          <a href="#">{{ buttonText }}</a>
+        </lg-card-content>
+      </lg-card>
+    </div>
+    <div lgColSm="12" lgColLg="4">
+      <lg-card
+        lgShadow
+        lgMarginBottom="lg"
+        variant="promotion"
+        [orientation]="{ sm: 'vertical', md: 'horizontal', lg: 'vertical' }">
+        <lg-card-hero-img>
+          <lg-brand-icon [name]="iconName" size="xs"></lg-brand-icon>
+        </lg-card-hero-img>
+        <lg-card-content>
+          <h3 lgMarginBottom="xxs" class="lg-font--expressive">{{ title }}</h3>
+          <p lgMarginBottom="lg">{{ text }}</p>
+          <a href="#">{{ buttonText }}</a>
+        </lg-card-content>
+      </lg-card>
+    </div>
+  </div>
+  <div lgRow>
+    <div lgCol="12">
+      <lg-separator variant="dotted" lgMarginTop="none"></lg-separator>
+    </div>
+    <div lgColSm="12" lgColLg="6">
+      <lg-card
+        lgShadow
+        lgMarginBottom="lg"
+        variant="promotion"
+        [orientation]="{ sm: 'vertical', md: 'horizontal', lg: 'vertical' }">
+        <lg-card-hero-img>
+          <lg-brand-icon [name]="iconName" size="xs"></lg-brand-icon>
+        </lg-card-hero-img>
+        <lg-card-content>
+          <h3 lgMarginBottom="xxs" class="lg-font--expressive">{{ title }}</h3>
+          <p lgMarginBottom="lg">{{ text }}</p>
+          <a href="#">{{ buttonText }}</a>
+        </lg-card-content>
+      </lg-card>
+    </div>
+    <div lgColSm="12" lgColLg="6">
+      <lg-card
+        lgShadow
+        lgMarginBottom="lg"
+        variant="promotion"
+        [orientation]="{ sm: 'vertical', md: 'horizontal', lg: 'vertical' }">
+        <lg-card-hero-img>
+          <lg-brand-icon [name]="iconName" size="xs"></lg-brand-icon>
+        </lg-card-hero-img>
+        <lg-card-content>
+          <h3 lgMarginBottom="xxs" class="lg-font--expressive">{{ title }}</h3>
+          <p lgMarginBottom="lg">{{ text }}</p>
+          <a href="#">{{ buttonText }}</a>
+        </lg-card-content>
+      </lg-card>
+    </div>
+  </div>
+</div>
+```
+## Additional development details
+
+### LgCardComponent
+
+The main body of the promotion card. It's composed by the following components:
+
+- `LgCardHeroImageComponent`
+- `LgCardContent`
+
+#### Inputs
+The orientation input arranges the direct children of the card in vertical or horizontal layout, a responsive way.
```

Comment:
Maybe I'm reading it wrong, should it be `..., in a responsive way`?

Reply:
Done, but moved to the Orientation Directive Guide. :)